OracleDatabase/RAC/OracleRealApplicationClusters/common/scripts/cmdExec

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+#############################
+# Copyright (c) 2018-2024 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please change all copyright lines for _new_ files to:

`Copyright (c) 2024, Oracle and/or its affiliates.`